Client/Ecosia/UI/WhatsNew/DataProvider/WhatsNewLocalDataProvider.swift

```diff
@@ -100,11 +100,20 @@ final class WhatsNewLocalDataProvider: WhatsNewDataProvider {
         evaluateVersionNeedsUpdate()
         
         // Ensure fromVersion is available.
-        guard let fromVersion else { return [] }
+        guard let fromVersion = self.fromVersion else { return [] }
+        
+        // If there's no update (i.e., the versions are the same), we shouldn't return any versions.
+        guard fromVersion != toVersion else { return [] }
         
         // Gather all versions
         let allVersions = Array(whatsNewItems.keys).sorted()
         
+        // Gather first item in `allVersions` array
+        guard let firstItemInAllVersions = allVersions.first else { return [] }
+        
+        // Ensure the `toVersion` is equal to or bigger than the smallest version in `whatsNewItems`
+        guard toVersion >= firstItemInAllVersions else { return [] }
```

Comment:
I guess this is sort of an edge case that should not happen from the moment we release the first version that has a matching item on the local provider, right?

Just wrapping my head around it since I spent some minutes going through the logic of this method in my head 😅 But the comments help 🙂

Reply:
Correct! 
On top, I wanted to have it as robust as possible so that we could theoretically not even care about the order at which we add items in the local provider.